weed/server/master_server.go

```diff
@@ -358,5 +361,27 @@ func (ms *MasterServer) OnPeerUpdate(update *master_pb.ClusterNodeUpdate, startF
 				hashicorpRaft.ServerID(peerName),
 				hashicorpRaft.ServerAddress(peerAddress.ToGrpcAddress()), 0, 0)
 		}
+	} else {
+		pb.WithMasterClient(false, peerAddress, ms.grpcDialOption, true, func(client master_pb.SeaweedClient) error {
+			ctx, cancel := context.WithTimeout(context.TODO(), time.Minute*72)
```

Comment:
why it is 72 minutes?

Reply:
I took it from RaftServerRemovalTime from line 37 https://github.com/kmlebedev/seaweedfs/commit/6cfbfb084941e213f38a83a763fd359cc6611108#diff-1f7b377a963eee3cdc38f3f4b7c168638a43855c5e3b03e7fcc00504dba4389aL37 . The reasoning behind this is to remove server that didn't recover for too long.
I can make this number configurable if you would like it this way

Reply:
Merged. 

@kmlebedev 
Why we need to waitForReady here? If the connection has problem, shouldn't we declare this server is disconnected? 

The 72 minute timeout seems also too long.

Reply:
> The 72 minute timeout seems also too long.

@chrislusf This period should be long enough to not have the effect of temporarily taking the server out for maintenance, such as kernel upgrades, disk replacement.

Reply:
It can be added back if it comes alive, right?

Reply:
> It can be added back if it comes alive, right?

Yes